### PR TITLE
Refactor mask with dynamic svg shape + Add rounded prop

### PR DESCRIPTION
**Pull request by qboot**
The mask is created with a combination of 4 `div` + 1 if `disableInteraction` is on.
Even if the actual situation is nearly perfect with default zoom, when we zoom-in we can see a little gap between `div` :

before

So, I have recoded it by using a single `svg` with `mask` and `clipPath` to recreate the same effect as before :

after

Plus, I have added a new `rounded` prop which allows you to apply a sort of `border-radius` in `px`. Default is `0`.

This PR needs maybe some more testing on different browsers / OS because of the use of css `pointer-events` property (supported: [94.88% on caniuse](https://caniuse.com/#feat=pointer-events)) and `svg` (supported: [97.88% on caniuse](https://caniuse.com/#feat=svg)).

### CODE REVIEW

**Comment by elrumordelaluz** [comment]
Absolutely in love with this PR!

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ const steps = [
 | `onBeforeClose` | function triggered before close | `func` | `() => { document.body.style.overflowY = 'auto' }`  |  |
 | `onRequestClose` | function triggered to close | `func` |  |  |
 | `prevButton` | prev navigation button text | `string` |  |  |
+| `rounded` | Beautify helper + mask with border-radius (in px) | `number` | `0` |  |
```
Comment (elrumordelaluz) [inline README.md]: Seems there's a conflict with missing props added in the last PR (`prevStep`, `nextStep`)

File: src/TourPortal.js [hunk 2]
```diff
@@ -58,6 +54,7 @@ class TourPortal extends Component {
     update: PropTypes.string,
     updateDelay: PropTypes.number,
     disableInteraction: PropTypes.bool,
+    rounded: PropTypes.number,
```
Comment (elrumordelaluz) [inline src/TourPortal.js]: also here with `disableNavigationDotsInteraction` and `disableKeyboardNavigation`, right?

**Comment by qboot** [comment]
Good catch ! 
Conflicts resolved, thanks for the review :grin:

**Comment by elrumordelaluz** [comment]
yup! thanks to you for this great new feature/improvement!

**Pull request merged**